bundles/org.openhab.binding.homekit/README.md

```diff
@@ -0,0 +1,149 @@
+# HomeKit Binding
+
+This binding allows pairing with HomeKit accessories and **imports** their services as channel groups and their respective service- characteristics as channels.
+Do not confuse this with the [HomeKit system integration](https://www.openhab.org/addons/integrations/homekit/) which **exports** openHAB Items to a HomeKit controller.
+
+## Supported Things
+
+There are three types of Things supported:
+
+- `accessory`: This integrates a single HomeKit accessory, which has its own LAN connection.
+  Its services appear as Channel Groups, and their respective characteristics appear as Channels.
+- `bridged-accessory`: This integrates a single HomeKit accessory, which does NOT have its own LAN connection.
+  It has the same functionality as an `accessory`, except that its communication is done via a `bridge` (see below).
+- `bridge`: This integrates a HomeKit bridge accessory, which has its own LAN connection.
+  It does not have any own Channels.
+  Instead it contains multiple `bridged-accessory` Things (see above).
+
+Things of type `bridge` and `accessory` both communicate directly with their HomeKit accessory device via the LAN.
+Whereas `bridged-accessory` Things communicate via their respective `bridge` Thing.
+
+## Discovery
+
+Both `accessory` and `bridge` Things will be auto-discovered via mDNS.
+And once a `bridge` Thing has been instantiated and paired, its `bridged-accessory` Things will also be auto-discovered.
+
+## Configuration for `bridge` and `accessory` Things
+
+The following table shows the Thing configuration parameters for `bridge` and `accessory` Things.
+
+| Name              | Type    | Description                                          | Default   | Required | Advanced |
+|-------------------|---------|------------------------------------------------------|-----------|----------|----------|
+| `ipAddress`       | text    | IP v4 address of the HomeKit accessory.              | see below | yes      | yes      |
+| `httpHostHeader`  | text    | The fully qualified host name as discovered by mDNS. | see below | yes      | yes      |
+| `macAddress`      | text    | Unique accessory identifier.                         | see below | yes      | yes      |
+| `refreshInterval` | integer | Interval at which the accessory is polled in sec.    | 60        | no       | yes      |
+
+NOTE: as a general rule, if you create the Things via the Inbox, then all of the above configuration parameters will have their proper values already preset.
```

Comment:
How about mentioning "mDNS" and "manual configuration" here, so it can be removed below?
```suggestion
NOTE: as a general rule, if you create the Things via the Inbox from the mDNS discovery result, then all of the above configuration parameters will have their proper values already preset.
However you can configure them manually if you wish.
```